docs/source/cheat_sheet_py3.rst

```diff
@@ -94,6 +94,36 @@ Python 3 introduces an annotation syntax for function declarations in `PEP 3107
            yield i
            i += 1
 
+   # A Python 3.4 coroutine should have a return type of
+   # Generator[Any, None, T], where T is the type it returns.
+   @asyncio.coroutine
```

Comment:
I think we try to keep the cheat sheet copy/paste runnable. Could you add an import for `asyncio`?

Reply:
Sure! I've fixed up the imports, and also fixed the duplicate names that got thrown up when typechecking, so this now executes cleanly and the only error I get is the expected error for `quux` that was already there.